### PR TITLE
issue 15: profileViewController mvp

### PR DESCRIPTION
**Pull request by saviotsui**
profileviewcontroller

### CODE REVIEW

File: ios-trivia-game/ViewControllers/CameraViewController.swift [hunk 1]
```diff
@@ -10,6 +10,14 @@ import UIKit
 
 class CameraViewController: UIViewController {
 
+    @IBAction func onBack(_ sender: UIBarButtonItem) {
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        let destination = storyboard.instantiateViewController(withIdentifier: "com.iostriviagame.maintabviewcontroller") as! MainTabViewController
+        destination.selectedIndex = 2
+        destination.navigationController?.isNavigationBarHidden = true
```
Comment (saviotsui) [inline ios-trivia-game/ViewControllers/CameraViewController.swift:17]: this is how i got the back button to stop displaying when i navigated back to to the profile view controller in the tabviewcontroller.

by default tabviewcontroller has its own navigationcontroller

File: ios-trivia-game/ViewControllers/MainTabViewController.swift [hunk 2]
```diff
@@ -14,6 +14,7 @@ class MainTabViewController: UITabBarController {
         super.viewDidLoad()
 
         // Do any additional setup after loading the view.
+        self.navigationController?.isNavigationBarHidden = true
```
Comment (saviotsui) [inline ios-trivia-game/ViewControllers/MainTabViewController.swift:17]: I ended up hiding the navigationbar completely. let me know if this is an issue for you two

File: ios-trivia-game/ViewControllers/ProfileViewController.swift [hunk 3]
```diff
@@ -7,13 +7,47 @@
...
-
-        // Do any additional setup after loading the view.
+        
+        self.user = User.currentUser
+        
+        self.profileImageView.clipsToBounds = true
+        self.profileImageView.setImageWith(URL(string: self.user.photoUrl!)!)
+        self.profileImageView.layer.cornerRadius = self.profileImageView.frame.size.width / 2;
+        
+        self.nicknameLabel.text = "somenickname"
```
Comment (saviotsui) [inline ios-trivia-game/ViewControllers/ProfileViewController.swift:48]: I'm thinking besides pulling data from FB, we might want to append to the profile and store it in firebase to complement what fb gives us.  Nickname for example

File: ios-trivia-game/ViewControllers/ProfileViewController.swift [hunk 4]
```diff
@@ -33,3 +67,33 @@ class ProfileViewController: UIViewController {
     */
 
 }
+
+extension UIImage {
```
Comment (saviotsui) [inline ios-trivia-game/ViewControllers/ProfileViewController.swift:71]: If others need this, I will move this to it's own file. Let me know

File: ios-trivia-game/ViewControllers/CameraViewController.swift (hunk 1 above)
Comment (wassupnari) [reply ios-trivia-game/ViewControllers/CameraViewController.swift:17]: Oh yeah I implemented similar way for the transition from CreateGameViewController to SelectFriendsViewController.

File: ios-trivia-game/ViewControllers/ProfileViewController.swift (hunk 4 above)
Comment (wassupnari) [reply ios-trivia-game/ViewControllers/ProfileViewController.swift:71]: I'm okay with current implementation. :)

**Comment by wassupnari** [comment]
Looks good! I'll merge this in!

**Pull request merged**